fiftyone/core/config.py

```diff
@@ -89,6 +106,24 @@ def __init__(self, d):
             env_var="FIFTYONE_DEFAULT_VIDEO_EXT",
             default=".mp4",
         )
+        self.default_app_color_pool = self.parse_string_array(
+            d,
+            "default_app_color_pool",
+            env_var="FIFTYONE_DEFAULT_APP_COLOR_POOL",
+            default=_COLOR_POOL,
+        )
+        self.default_app_show_confidence = self.parse_bool(
+            d,
+            "default_app_show_confidence",
+            env_var="FIFTYONE_DEFAULT_APP_SHOW_CONFIDENCE",
```

Comment:
w.r.t. the `AppConfig` proposal above, perhaps the App-config-related variables should namespaced into a `fo.config.app_config` variable?

But the `EnvConfig` class doesn't handle parameters that are objects, not sure what the cleanest way to proceed there is... 🤔

Reply:
I think a `default_app_config_path` will do. I'll write an `AppConfig` tomorrow AM.